tmt/steps/provision/bootc.py

```diff
@@ -271,6 +271,8 @@ def _build_bootc_disk(self, containerimage: str, image_builder: str) -> None:
             "build",
             "--type",
             "qcow2",
+            "--rootfs",
+            "xfs",
```

Comment:
Thanks for working on this! I think it would fix the problem, but not sure if `centos-bootc` needs this.

Reply:
Would it make sense to make the addition and/or the FS type configurable?

Reply:
yes, it would be good to make the filesystem type a configuration option

Reply:
we can probably also duplicate the test to make sure that we test both fedora & centos bootc images here.

Reply:
I guess,  to address @travier's  request you need:
Extend https://github.com/teemtee/tmt/blob/main/tests/provision/bootc/test.sh and add two more files,
say, includes-deps-fedora.containerfile  needs-deps-fedora.containerfile.

Otherwise,looks good to me.

Reply:
Hi @skycastlelily, in this case `includes-deps.containerfile` will cover `fedora-bootc:41` and `needs-deps.containerfile` covers `centos-bootc:stream9`. We have all of them covered. So do not need extra test added.

Reply:
TBO, I got your thoughts before I commented, and I'm not quite familiar with bootc stuff, but I believe the original contributor @ckyrouac add both includes-deps and needs-deps tests  for some reason, and your commit would will not test needs-deps for fedora and includes-deps for centos.

Reply:
Right. The `includes-deps` is for Containerfile which has `dnf install rsync cloud-init` (tmt needs) installed. The `needs-deps` is for Containerfile which does not have `dnf install rsync cloud-init` installed and to test `add-tmt-dependencies` property.

So one for rhel-bootc and another for fedora-bootc don't have any impact for the test, but add one more image covered.